<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TweezerWindow</class>
 <widget class="QMainWindow" name="TweezerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>751</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="current_trap_forks_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Current TrapForks</string>
    </property>
   </widget>
   <widget class="ImageView" name="image_plot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>451</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageView" name="fork_plots_trap">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="data_to_fetch">
     <item>
      <widget class="QLabel" name="pos_label">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pos_no_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="trap_no_label">
       <property name="text">
        <string>Trap No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="trap_no_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="fetch_button">
       <property name="text">
        <string>Fetch</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="data_to_plot">
     <item>
      <widget class="QRadioButton" name="phase_image">
       <property name="text">
        <string>Phase</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cell_seg_image">
       <property name="text">
        <string>Cell Seg</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fluor_image">
       <property name="text">
        <string>Fluor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dots_on_mask_image">
       <property name="text">
        <string>Dots on mask</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="image_show_layout">
     <item>
      <widget class="QRadioButton" name="get_last20_radio">
       <property name="text">
        <string>Last 20 images</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="get_all_images_radio">
       <property name="text">
        <string>All Images</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="ImageView" name="barcode_plot_1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>51</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageView" name="barcode_plot_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>51</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fork_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trap fork plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="barcodes_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Barcodes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_data_forks_button">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>80</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>All data forks</string>
    </property>
   </widget>
   <widget class="ImageView" name="fork_plots_all">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>140</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fork_label_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>120</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>All data fork plot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1296</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
